.rubocop_todo.yml

```diff
@@ -231,41 +231,11 @@ Layout/SpaceInsideReferenceBrackets:
     - 'test/remote/gateways/remote_ideal_rabobank_test.rb'
     - 'test/remote/gateways/remote_wirecard_test.rb'
 
-# Offense count: 3
-# Cop supports --auto-correct.
-# Configuration parameters: EnforcedStyle.
-# SupportedStyles: space, no_space
-Layout/SpaceInsideStringInterpolation:
-  Exclude:
-    - 'lib/active_merchant/billing/gateways/garanti.rb'
-    - 'lib/active_merchant/billing/gateways/trust_commerce.rb'
-    - 'test/unit/gateways/worldpay_test.rb'
-
-# Offense count: 4
-Lint/AmbiguousBlockAssociation:
-  Exclude:
-    - 'test/test_helper.rb'
-    - 'test/unit/gateways/gateway_test.rb'
-    - 'test/unit/gateways/worldpay_test.rb'
-
-# Offense count: 31
-Lint/AmbiguousRegexpLiteral:
-  Enabled: false
-
 # Offense count: 148
 # Configuration parameters: AllowSafeAssignment.
 Lint/AssignmentInCondition:
   Enabled: false
 
-# Offense count: 8
-Lint/DuplicateMethods:
```

Comment:
_In general_, for this one, I deleted the first method definition, since the latter would currently be superseding it, but I did audit the deletions to make sure that was a reasonable course of action. Feel free to look and make sure you agree.